rpcHub/rpcHub.go

```diff
@@ -157,3 +165,54 @@ func (self *RpcClient) Call(serviceMethod string, args interface{}, reply interf
 
 	return err
 }
+
```

Comment:
Is this code being replicated here from netutils just to avoid dependency on netplugin?